modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,209 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple clusters, thus helping to avoid a single point of failure. 
+
+[NOTE]
+====
+{product-title} vSphere topology awareness does not support multiple vCenter datacenters.
```

Comment:
err -  this reads bit awkwardly. We should just say:

```
{product-tile} on vSphere does not yet support multiple Datacenters.
```

Reply:
Done.